.github/workflows/ci.yaml

```diff
@@ -0,0 +1,76 @@
+name: CI
+
+on:
+    pull_request:
+    push:
+        branches:
+            - 'master'
+
+jobs:
+    php:
+        name: "PHP ${{ matrix.php-version }}"
+        runs-on: ubuntu-latest
+
+        env:
+            KERNEL_CLASS: DTL\Bundle\PhpcrMigrations\Tests\Resources\App\AppKernel
+
+        strategy:
+            fail-fast: false
+            matrix:
+                include:
+                    - php-version: '7.0'
```

Comment:
does Sulu need 7.0 support? think it would be OK to support 7.4+ only otherwise

Reply:
Currently we are on `^7.2 || ^8.0`. So would like to keep that.